cores/nRF5/Print.cpp

```diff
@@ -22,6 +22,7 @@
 #include <string.h>
 #include <math.h>
 #include "Arduino.h"
+#include "printf.h"
```

Comment:
Let's just change this to `#include "libc/printf/printf.h"` and then we don't need to platform update and will also fix Arduino builds.